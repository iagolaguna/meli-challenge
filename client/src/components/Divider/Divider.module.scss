@import "styles/colors";

.divider {
  width: 100%;
  height: 2px;
  background-color: $light_gray;
}
