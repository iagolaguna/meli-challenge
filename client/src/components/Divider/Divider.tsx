import style from "./Divider.module.scss";

const Divider = () => {
  return <div className={style.divider} />;
};

export default Divider;
