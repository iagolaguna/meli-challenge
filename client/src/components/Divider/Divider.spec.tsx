import { render } from "@testing-library/react";
import Divider from "components/Divider/Divider";

describe("<Divider/>", () => {
  it("should match snapshot", () => {
    const { container } = render(<Divider />);
    expect(container).toMatchSnapshot();
  });
});
