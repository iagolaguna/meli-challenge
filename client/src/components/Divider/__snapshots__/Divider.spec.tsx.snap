// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider/> should match snapshot 1`] = `
<div>
  <div
    aria-hidden="false"
    class="divider"
  />
</div>
`;
